doc/audit/index.rst

```diff
@@ -3,6 +3,13 @@ Audit
 
 .. panels::
 
+    :fa:`scroll` :doc:`receipts`
```

Comment:
Should this only be included once, under `audit/` rather than under `architecture/` too?